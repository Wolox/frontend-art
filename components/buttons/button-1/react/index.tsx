import React from 'react';

import '../styles.scss';

function Button1() {
  return (
    <div className="column center btn-group-1">
      <button className="btn btn-1 m-bottom-2" disabled={true}>
        <div className="btn__content">cancel</div>
      </button>
      <button type="button" className="btn btn-1">
        <div className="btn__content">submit</div>
      </button>
    </div>
  );
}

export default Button1;
