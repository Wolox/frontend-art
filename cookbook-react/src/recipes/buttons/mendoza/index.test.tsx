import React from 'react';
import { render } from '@testing-library/react';

import ButtonExample from './index';

test('render component', () => {
  const { container } = render(<ButtonExample />);
  expect(container).toMatchSnapshot();
});
