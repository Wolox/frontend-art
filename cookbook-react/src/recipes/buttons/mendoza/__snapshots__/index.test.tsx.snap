// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render component 1`] = `
<div>
  <div
    class="column center"
  >
    <button
      class="button primary m-bottom-2"
      type="button"
    >
      Primary
    </button>
    <button
      class="button primary m-bottom-2"
      disabled=""
      type="button"
    >
      Primary Disabled
    </button>
    <button
      class="button secondary m-bottom-2"
      type="button"
    >
      Secondary
    </button>
    <button
      class="button secondary"
      disabled=""
      type="button"
    >
      Secondary Disabled
    </button>
  </div>
</div>
`;
